keys-and-keywords/ubuntu-keywords.robot

```diff
@@ -40,3 +40,22 @@ Install Package
     Write Into Terminal    apt-get install --assume-yes ${package}
     Read From Terminal Until Prompt
     Set DUT Response Timeout    30s
+
+Stop Logging To Terminal
+    [Documentation]    This keyword stops all unwanted logging to the terminal
+    ...    we are using for the shell. This prevents garbage from being logged.
+    Set Logging Level    0
+
+Get Logging Level
+    [Documentation]    This keyword returns TRUE if logging is disabled and
+    ...    FALSE if it is not.
+    ${ret}=    Execute Linux Command    cat /proc/sys/kernel/printk | cut -f1
+
+    RETURN    ${ret}
+
+Set Logging Level
+    [Documentation]    This keyword sets the logging level to given value [0; 7]
+    [Arguments]    ${level}
+    Execute Linux Command    echo "kernel.printk = ${level} 4 1 7" > /etc/sysctl.d/10-console-messages.conf
+    Execute Linux Command    echo "kernel.printk = ${level} 4 1 7" >> /etc/sysctl.conf
+    Execute Linux Command    sysctl --system
```

Comment:
this could be changed to oneliner too: 
```echo "kernel.printk = ${level} 4 1 7" | tee /etc/sysctl.d/10-console-messages.conf >> /etc/sysctl.conf && sysctl --system```

however I leave it for your consideration.

Reply:
Thanks. I am not sure which version would be better though. If performance is considered then the oneliner should be preferred. If not then separating this into three calls may be a bit more readable in the code and in the logs?

Reply:
Turns out 4 actions need to be performed here, this could, again, be joined into one `Execute Linux Command` call, but in my opinion keeping these actions separate would be better.
 
https://github.com/Dasharo/open-source-firmware-validation/pull/449/commits/c2910d941d4d01840e0e8479d6c1dbaa8eb8f728#diff-b7afbf080f275d1995a5212ce98c26d58fb1004705c786cc31496e1043ac04b6R59-R62